zerver/lib/exceptions.py

```diff
@@ -174,7 +174,7 @@ def __init__(self, stream: str) -> None:
     @staticmethod
     @override
     def msg_format() -> str:
-        return _("Stream '{stream}' does not exist")
+        return _("Channel '{stream}' does not exist")
```

Comment:
As noted in the commit message, the variable here, and in the ID version of this error, can't be updated without impacting clients as it would change the name of the field sent in the error response.